src/as-content-rating.h

```diff
@@ -153,6 +160,12 @@ void			as_content_rating_add_attribute (AsContentRating *content_rating,
 							const gchar *id,
 							AsContentRatingValue value);
 
+gboolean	as_is_oars_key (const gchar *id, AsOarsVersion version);
+
+AsOarsVersion	as_oars_version_from_string (const gchar *value);
+
+gboolean	as_content_rating_value_is_valid (const gchar *id, AsContentRatingValue value);
```

Comment:
Please don't add those as public API, make them internal API instead. Put functions like this in `as-content-rating-private.h` and you should be fine, same for the enums.